src/components/Reels.jsx

```diff
@@ -64,6 +64,16 @@ export default function Reels() {
             <Slot index={index} slotIndex={slotIndex} items={items} />
           ))}
         </StyledReel>
+        <StyledReel>
+          {reel3.map((slotIndex, index) => (
+            <Slot index={index} slotIndex={slotIndex} items={items} />
+          ))}
+        </StyledReel>
+        <StyledReel>
+          {reel3.map((slotIndex, index) => (
+            <Slot index={index} slotIndex={slotIndex} items={items} />
```

Comment:
since you are using `.map`, `Slot` will need a `key` for react to track array of components.

https://legacy.reactjs.org/docs/lists-and-keys.html#keys